<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainMenu</class>
 <widget class="QWidget" name="MainMenu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#RootFrame , #ButtonSection, #ContentFrame, #SearchFrame{
background-color: qlineargradient(spread:pad, x1:0, y1:0.522273, x2:1, y2:0.614, stop:0 rgba(0, 0, 0, 255), stop:0.25 rgba(59, 59, 59, 255), stop:0.456731 rgba(0, 0, 0, 255), stop:0.697115 rgba(61, 61, 61, 255), stop:1 rgba(0, 0, 0, 255));
border: solid;
border-width: 2;
border-radius: 10;
border-color: qlineargradient(spread:pad, x1:0.0240385, y1:0.54, x2:1, y2:0.551, stop:0 rgba(156, 156, 156, 255), stop:0.153846 rgba(104, 104, 104, 255), stop:0.360577 rgba(187, 187, 187, 255), stop:0.725962 rgba(105, 105, 105, 255), stop:0.942308 rgba(132, 132, 132, 255));
margin: 5;
}</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QFrame" name="RootFrame">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_2" rowstretch="0,1,0">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <property name="spacing">
       <number>0</number>
      </property>
      <item row="2" column="0">
       <widget class="QFrame" name="SearchFrame">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QGridLayout" name="gridLayout_6">
         <item row="0" column="1">
          <widget class="QLineEdit" name="lineEdit"/>
         </item>
         <item row="0" column="2">
          <widget class="QQuickWidget" name="Search_BTN">
           <property name="minimumSize">
            <size>
             <width>175</width>
             <height>40</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>175</width>
             <height>40</height>
            </size>
           </property>
           <property name="resizeMode">
            <enum>QQuickWidget::SizeRootObjectToView</enum>
           </property>
          </widget>
         </item>
         <item row="0" column="3">
          <widget class="QQuickWidget" name="ResetSearch_BTN">
           <property name="minimumSize">
            <size>
             <width>175</width>
             <height>40</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>175</width>
             <height>40</height>
            </size>
           </property>
           <property name="resizeMode">
            <enum>QQuickWidget::SizeRootObjectToView</enum>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QComboBox" name="comboBox"/>
         </item>
        </layout>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QFrame" name="ApplicationControlFrame">
        <property name="styleSheet">
         <string notr="true">QFrame {
border: none;
background-color: transparent;

}

#ButtonSection {
background-color: qlineargradient(spread:pad, x1:0, y1:0.522273, x2:1, y2:0.614, stop:0 rgba(0, 0, 0, 255), stop:0.25 rgba(59, 59, 59, 255), stop:0.456731 rgba(0, 0, 0, 255), stop:0.697115 rgba(61, 61, 61, 255), stop:1 rgba(0, 0, 0, 255));
border: solid;
border-width: 2;
border-radius: 10;
border-color: qlineargradient(spread:pad, x1:0.0240385, y1:0.54, x2:1, y2:0.551, stop:0 rgba(156, 156, 156, 255), stop:0.153846 rgba(104, 104, 104, 255), stop:0.360577 rgba(187, 187, 187, 255), stop:0.725962 rgba(105, 105, 105, 255), stop:0.942308 rgba(132, 132, 132, 255));
margin: 5;
}</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QGridLayout" name="gridLayout_3" rowstretch="0,0,0">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>5</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <property name="spacing">
          <number>0</number>
         </property>
         <item row="1" column="0">
          <widget class="QFrame" name="TitleSection">
           <property name="styleSheet">
            <string notr="true">QFrame {
border: none;
background-color: transparent;
}</string>
           </property>
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QGridLayout" name="gridLayout_4">
            <item row="0" column="1">
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item row="0" column="0">
             <widget class="QLabel" name="label">
              <property name="font">
               <font>
                <pointsize>10</pointsize>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">QLabel {
color: white;

}</string>
              </property>
              <property name="text">
               <string>Publication Organizer</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
              </property>
             </widget>
            </item>
            <item row="0" column="2">
             <widget class="QQuickWidget" name="Minimize_BTN">
              <property name="minimumSize">
               <size>
                <width>35</width>
                <height>22</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>35</width>
                <height>22</height>
               </size>
              </property>
              <property name="resizeMode">
               <enum>QQuickWidget::SizeRootObjectToView</enum>
              </property>
             </widget>
            </item>
            <item row="0" column="3">
             <widget class="QQuickWidget" name="MaximizeRestore_BTN">
              <property name="minimumSize">
               <size>
                <width>35</width>
                <height>22</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>35</width>
                <height>22</height>
               </size>
              </property>
              <property name="resizeMode">
               <enum>QQuickWidget::SizeRootObjectToView</enum>
              </property>
             </widget>
            </item>
            <item row="0" column="4">
             <widget class="QQuickWidget" name="Close_BTN">
              <property name="minimumSize">
               <size>
                <width>35</width>
                <height>22</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>35</width>
                <height>22</height>
               </size>
              </property>
              <property name="resizeMode">
               <enum>QQuickWidget::SizeRootObjectToView</enum>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QFrame" name="ButtonSection">
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QGridLayout" name="gridLayout_5">
            <item row="0" column="2">
             <widget class="QQuickWidget" name="EditPublication_BTN">
              <property name="minimumSize">
               <size>
                <width>175</width>
                <height>40</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>175</width>
                <height>40</height>
               </size>
              </property>
              <property name="resizeMode">
               <enum>QQuickWidget::SizeRootObjectToView</enum>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QQuickWidget" name="AddPublication_BTN">
              <property name="minimumSize">
               <size>
                <width>175</width>
                <height>40</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>175</width>
                <height>40</height>
               </size>
              </property>
              <property name="resizeMode">
               <enum>QQuickWidget::SizeRootObjectToView</enum>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QQuickWidget" name="DeletePublication_BTN">
              <property name="minimumSize">
               <size>
                <width>175</width>
                <height>40</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>175</width>
                <height>40</height>
               </size>
              </property>
              <property name="resizeMode">
               <enum>QQuickWidget::SizeRootObjectToView</enum>
              </property>
             </widget>
            </item>
            <item row="0" column="3">
             <widget class="QQuickWidget" name="PrintReport_BTN">
              <property name="minimumSize">
               <size>
                <width>175</width>
                <height>40</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>175</width>
                <height>40</height>
               </size>
              </property>
              <property name="resizeMode">
               <enum>QQuickWidget::SizeRootObjectToView</enum>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QFrame" name="ContentFrame">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QGridLayout" name="gridLayout_7">
         <item row="0" column="0">
          <widget class="QTableWidget" name="tableWidget">
           <property name="styleSheet">
            <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0.522273, x2:1, y2:0.614, stop:0 rgba(0, 0, 0, 255), stop:0.25 rgba(59, 59, 59, 255), stop:0.456731 rgba(0, 0, 0, 255), stop:0.697115 rgba(61, 61, 61, 255), stop:1 rgba(0, 0, 0, 255));</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QQuickWidget</class>
   <extends>QWidget</extends>
   <header location="global">QtQuickWidgets/QQuickWidget</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
